infra/gcb/build_and_run_coverage.py

```diff
@@ -44,8 +44,8 @@
 # Link where to upload code coverage report files to.
 UPLOAD_URL_FORMAT = 'gs://' + COVERAGE_BUCKET_NAME + '/{project}/{type}/{date}'
 
-# TODO(#2817): gofuzz projects to skip code coverage job for.
-GO_FUZZ_PROJECTS = ['golang', 'syzkaller']
+# TODO(#2817): code coverage is not supported for Go projects.
```

Comment:
nit: Rephrase as "Support code coverage for Go projects).

Reply:
Done